build.sbt

```diff
@@ -3,78 +3,97 @@ import scalariform.formatter.preferences._
 
 organization in ThisBuild := "net.ruippeixotog"
 
-scalaVersion in ThisBuild := "2.12.8"
-crossScalaVersions in ThisBuild := Seq("2.11.12", "2.12.8")
```

Comment:
Is there any reason why you removed Scala 2.11? Did any library stop supporting it?

Reply:
Http4s does not seem to support it anymore